src/bin/vip-config-envvar-list.js

```diff
@@ -57,13 +57,18 @@ export async function listEnvVarsCommand( arg: string[], opt ): void {
 		process.exit();
 	}
 
-	// Display context for non-machine-readable formats.
-	if ( [ 'keyValue', 'table' ].includes( opt.format ) ) {
-		console.log( 'For security, the values of environment variables cannot be retrieved.' );
-		console.log();
+	// Vary data by expected format.
+	let key = 'name';
+	if ( 'keyValue' === opt.format ) {
```

Comment:
`keyValue` and `ids` doesn't seem to make as much sense. I liked `table`, and `csv` sounds useful, but when I try `--format=csv` I get basically a table without the borders. Is that expected?

Reply:
Yeah, I agree, with a single column of data, some of these formats aren't very useful. If we're thinking about the use case of piping this data into another script (like Node or Bash or Python), then I think `ids` and `csv` are probably the most useful.

`keyValue` really doesn't make much sense. But the `format` arg is shared among lots of commands and isn't configurable. Maybe I'll look into a way to customize it in a separate PR.